metricbeat/module/redis/info/data.go

```diff
@@ -234,15 +236,33 @@ var (
 	}
 )
 
+func buildCommandstatsSchema(key string, schema s.Schema) {
+	// Build schema for each command
+	command := strings.Split(key, "_")[1]
+	schema[command] = s.Object{
+		"calls":         c.Int("cmdstat_" + command + "_calls"),
+		"usec":          c.Int("cmdstat_" + command + "_usec"),
+		"usec_per_call": c.Float("cmdstat_" + command + "_usec_per_call"),
+	}
+}
+
 // Map data to MapStr
 func eventMapping(r mb.ReporterV2, info map[string]string) {
 	// Full mapping from info
 	source := map[string]interface{}{}
+	commandstatsSchema := s.Schema{}
 	for key, val := range info {
 		source[key] = val
+		if strings.Contains(key, "_calls") {
```

Comment:
minor: `strings.HasPrefix(key, "cmdstat_")` is probably less prone to error.
the drawback is that it will return true multiple times. given that `schema[command]` in `buildCommandstatsSchema` will always be the same in can be a minor drawback